### PR TITLE
Jacob user controller

### PR DESCRIPTION
**Pull request by CJacobJ**
Adds UserController functions:
createUser
updateUser
getUser

Creates UserEdit object to store incoming edit requests.

Adds validationService function validateJwt(ReverseJWT jwt, long userID)

### CODE REVIEW

**Comment by ryanmohler17** [comment]
jenkins build

**Comment by ryanmohler17** [comment]
jenkins build

**Comment by ryanmohler172** [comment]
[61% (-20.11%) vs master 81%](http://3.91.248.52:8080/jenkins/job/User-Service%20-%20PR/21/)

**Comment by CJacobJ** [comment]
I'm working on the merge conflicts right now, that's why its a draft.

**Comment by ryanmohler172** [comment]
[81% (+0.19%) vs master 81%](http://3.91.248.52:8080/jenkins/job/User-Service%20-%20PR/22/)

**Comment by CJacobJ** [comment]
Ready for review!

**Comment by ryanmohler172** [comment]
[81% (-0.45%) vs master 81%](http://3.91.248.52:8080/jenkins/job/User-Service%20-%20PR/23/)

File: src/main/java/com/reverse/userservice/controllers/UserController.java [hunk 1]
```diff
@@ -1,44 +1,80 @@
...
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping(path = "user")
-@Component("UserController")
+@RequestMapping(path = "users")
 public class UserController {
 
+    private Logger logger = LoggerFactory.getLogger(UserController.class);
```
Comment (ARosenbaum98) [inline src/main/java/com/reverse/userservice/controllers/UserController.java:26]: Look into using @slf4j so you can avoid instantiating your loggers : )

**Review by ARosenbaum98 — APPROVED** [review]

**Pull request merged**